<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_SystemSettings</class>
 <widget class="QFrame" name="F_SystemSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_42">
   <item>
    <widget class="QPlainTextEdit" name="text">
     <property name="plainText">
      <string>AD	AND	020	AN	Andorra	Andorra la Vella	468	84000	EU	.ad	EUR	Euro	376	AD###	^(?:AD)*(\d{3})$	ca	3041565	ES,FR	
AE	ARE	784	AE	United Arab Emirates	Abu Dhabi	82880	4975593	AS	.ae	AED	Dirham	971			ar-AE,fa,en,hi,ur	290557	SA,OM	
AF	AFG	004	AF	Afghanistan	Kabul	647500	29121286	AS	.af	AFN	Afghani	93			fa-AF,ps,uz-AF,tk	1149361	TM,CN,IR,TJ,PK,UZ	</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="L_info">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_41">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ISO</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ISO"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>ISO3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ISO3"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>ISO-Numeric</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ISO_Numeric"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_24">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>fips</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fips"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_25">
         <item>
          <widget class="QLabel" name="labelxx">
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Country"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_26">
         <item>
          <widget class="QLabel" name="Capitalff">
           <property name="text">
            <string>Capital</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Capital"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_27">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Area</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Area"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_28">
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Population</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Population"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_29">
         <item>
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Continent</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Continent"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_30">
         <item>
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>tld</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tld"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_31">
         <item>
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>CurrencyCode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="CurrencyCode"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_32">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>CurrencyName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="CurrencyName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_33">
         <item>
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>Phone</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Phone"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_34">
         <item>
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>Postal Code Format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PostalCodeFormat"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_35">
         <item>
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>Postal Code Regex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PostalCodeReg"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_36">
         <item>
          <widget class="QLabel" name="label_36">
           <property name="text">
            <string>Languages</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Languages"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_37">
         <item>
          <widget class="QLabel" name="label_37">
           <property name="text">
            <string>geonameid</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="geonameid"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_38">
         <item>
          <widget class="QLabel" name="label_38">
           <property name="text">
            <string>neighbours</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="neighbours"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_39">
         <item>
          <widget class="QLabel" name="label_39">
           <property name="text">
            <string>EquivalentFipsCode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="EquivalentFipsCode"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_40">
         <item>
          <widget class="QLabel" name="label_40">
           <property name="text">
            <string>Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="Cb_NumLine"/>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_GetLines">
       <property name="text">
        <string>Get Lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_SplitLine">
       <property name="text">
        <string>Split ONE LINE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
